go.mod

```diff
@@ -5,7 +5,7 @@ go 1.22.0
 toolchain go1.22.3
 
 require (
-	github.com/cloud-bulldozer/go-commons v1.0.16
+	github.com/cloud-bulldozer/go-commons v1.0.17
```

Comment:
This new version will help us detect indexing issues like below in early stages.
```
time="2024-10-30 08:42:35" level=info msg="Indexing metric nodeReadyLatencyMeasurement" file="common.go:28"
time="2024-10-30 08:42:35" level=info msg="Failed to index document with ID 9d1dfbf1aea95deafbae17735b12b1bf736733b567e71934054f24d1f0947065: Could not dynamically add mapping for field [node.kubernetes.io/instance-type]. Existing mapping for [labels.node] must be of type object but found [text]., error: <nil>" file="elastic.go:127"
time="2024-10-30 08:42:35" level=info msg="Failed to index document with ID 1a38180e64ee3b21f81ad8bccb790671f38807a1e330236038d743c52e3406f1: Could not dynamically add mapping for field [node.kubernetes.io/instance-type]. Existing mapping for [labels.node] must be of type object but found [text]., error: <nil>" file="elastic.go:127"
time="2024-10-30 08:42:35" level=info msg="Failed to index document with ID b1c0eee3dad9b67082e0ae3b9e1e5a96219b8e9bd7b831bd58129cb92d76080e: Could not dynamically add mapping for field [node.kubernetes.io/instance-type]. Existing mapping for [labels.node] must be of type object but found [text]., error: <nil>" file="elastic.go:127"
time="2024-10-30 08:42:35" level=info msg="Indexing finished in 261ms:" file="common.go:37"
```

Previously it just use to silently fail without proper explanation on what went wrong as show below.
```
time="2024-10-30 08:42:35" level=info msg="Indexing finished in 261ms:" file="common.go:37"
```
cc : @jtaleric

Reply:
This seems like a good fix, but why is it included in a PR that mentions `Enhancements for worker-scale metrics
` seems like a good thing to have regardless of worker-scale?